### DISCUSSION TITLE
YouTube ads since December 12

### DISCUSSION THREAD

**Discussion by chr7** [uBlockOrigin/uBlock-issues#3484] [Q&A]
It seems that YouTube prepared a little Christmas gift and has changed something since December 12 because suddenly videos stop after 1 minute. I have to disable uBO to continue (and get ads of course). Anybody else facing similar problems now?

**Comment by stephenhawk8054** [comment]
https://github.com/uBlockOrigin/uAssets/issues/20586#issuecomment-2539605188
